m4_sessions/1708-inning/group_four.md

```diff
@@ -1,9 +1,35 @@
  # Group Four
 
-### Session Title:
+### Session Title: 
+Privacy: What it means for devs and their users
+
 
 ### Session Outcomes: 
+- Students are aware of broader privacy concern implications
+- Students can make informed choices about privacy impacts for their users
+
 
 ### Session Outline:
+*This session is about privacy in the scope of Corporate snooping, not Government Surveillance. We're limiting our scope to allow more targeted discussions.
+
+- Intros
+
+- Activities
+* short video about creepy stuff?
+* maybe use my creepy app?
```

Comment:
Divide and conquer--maybe you could each find one video that you think is creepy.  Keep this to about 5-10 minutes at most.